JuiceMaker/JuiceMaker/Model/FruitStore.swift

```diff
@@ -1,12 +1,37 @@
 //
 //  JuiceMaker - FruitStore.swift
-//  Created by yagom. 
+//  Created by riji, kaki. 
 //  Copyright © yagom academy. All rights reserved.
 //
 
-import Foundation
-
 // 과일 저장소 타입
 class FruitStore {
     
+    private var fruitStocks: [Fruits: Int] = [:]
+    private let initialStock = 10
```

Comment:
```suggestion
class FruitStore {
    private var fruitStocks: [Fruits: Int] = [:]
    private let initialStock = 10
```

원래 한 줄 개행이 컨벤션인 걸까요~? 아니라면 맞춰주세요!

Reply:
개행 수정하였습니다!